web/templates/page/index.html.eex

```diff
@@ -1,11 +1,51 @@
-<%= if @current_user do %>
-  <div>
-    <h2>Welcome <%= @current_user.name %>!</h2>
-    <%= link "Logout", to: "/auth/logout", method: :delete %>
+<header class="">
+  <h1 class="tc">Help Wanted</h1>
+  <div class="f6 pa2 tc">
+    <p class=""> Help Wanted is a dwyl product, made with ❤️ , so that you can easily contribute to open source projects. Each of the issues below requires help from someone like you; technical or not.</p>
+    <p class=""> Take a look at our <a href="https://github.com/dwyl/labels#labels" target="_blank">labels repo</a> to familiarise yourself with what they mean and if you want to learn more about how to contribute to dwyl repos, check out our <a href="https://github.com/dwyl/contributing" target="_blank">contributing guide</a>.</p>
+    <p class="">We'll be adding features over time based on your recommendations so please leave us feedback through an issue and apply the <code>enhancement</code> label.</p>
   </div>
-<% else %>
-  <a class="dib link mv2 pa3 pa3-ns dwyl-bg-mint white br2 f4 shadow-4" href="/auth/github">
-    <i class="fa fa-github"></i>
-    Sign in with GitHub
-  </a>
-<% end %>
+</header>
+<section class="tl center ph2 dwyl-dark-gray">
+  <table class="w-80 center dn db-ns">
+    <thead>
+      <tr class="bb bw2 dwyl-b--gray">
+        <th class="pv3 pl2">Repo</th>
+        <th class="pv3 pl3">Title</th>
+        <th class="pv3">Assignees</th>
+      </tr>
+    </thead>
+    <tbody>
+      <%= for issue <- @issues do %>
```

Comment:
Typically I prefer to take out the information I'll need at this level by pattern matching in the for loop like so:

```elixir
<%= for %{"url" => url,
          "repo_name" => repo_name,
          "comments_number" => comments_number} <- @issues do %>
```

I find this nicer than `issue."url"`, not a big deal, just my preference